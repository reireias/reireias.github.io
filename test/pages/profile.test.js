import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import target from '@/pages/profile'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)

describe('profile', () => {
  it('should be vue instance', () => {
    const wrapper = shallowMount(target, {
      localVue,
      Vuetify,
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
