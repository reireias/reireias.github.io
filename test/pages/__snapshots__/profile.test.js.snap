// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile should be vue instance 1`] = `
"<v-container-stub tag=\\"div\\" class=\\"profile-container\\">
  <v-row-stub tag=\\"div\\" justify=\\"center\\">
    <div class=\\"display-3\\">Profile</div>
  </v-row-stub>
  <v-row-stub tag=\\"div\\" justify=\\"center\\">
    <v-col-stub cols=\\"12\\" md=\\"8\\" tag=\\"div\\">
      <profile-stub></profile-stub>
    </v-col-stub>
  </v-row-stub>
  <v-row-stub tag=\\"div\\" justify=\\"center\\">
    <v-col-stub cols=\\"12\\" md=\\"8\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" flat=\\"true\\">
        <v-card-title-stub class=\\"bottom-border-accent\\"> Accounts </v-card-title-stub>
        <v-container-stub tag=\\"div\\"></v-container-stub>
      </v-card-stub>
    </v-col-stub>
  </v-row-stub>
</v-container-stub>"
`;
